.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
+    # Check the npm registry for updates every day (weekdays)
```

Comment:
~~`# Check the npm registry for updates every day (weekdays)`~~

Reply:
Whoops. Fixed, thanks!